final-project/manJoinProcess.tex

```diff
@@ -63,4 +63,16 @@ \subsection{Messaging Protocol}
 
 \section{Creating or Loading Rooms}
 
+The client will have two room creation options: generating a temporary room with a random token and establishing a permanent room that can be rejoined in the future. 
```

Comment:
Although not accessible via the web GUI, API clients are able to create named temporary rooms (again, this is different from generating a room). So, you should distinguish the difference between "generating" and "creating", alongside "permanent" and "temporary".

Reply:
Just to ensure I understand this correctly: there are 3 room options. Generating is for random rooms with no inputs, creating is for rooms with inputs. Permanent rooms can only be created and temporary rooms can be either generated or created depending on if you wish to name them. Did I get that right?

Reply:
That's correct.

What this look like on the REST API is:
- `/api/room/generate` generates temporary rooms
- `/api/room/create` creates rooms based on provided options, one of them being `temporary=true` or `temporary=false`

Reply:
Thanks, will reflect these changes in the description as well as the sequence diagram.